ambry-router/src/main/java/com.github.ambry.router/GetOperation.java

```diff
@@ -44,8 +44,8 @@
   protected final Callback<T> operationCallback;
   protected final BlobId blobId;
   protected final Time time;
-  protected boolean operationCompleted = false;
-  protected Exception operationException;
+  protected volatile boolean operationCompleted = false;
```

Comment:
This gets set in an `abort()` call after the operation callback is invoked. You can see that for `GetBlobOperation#abort()`. Will set that for `GetBlobInfoOperation#abort()` as well.

Though it may be okay now to have that as non-volatile in the current code, it seemed important at the time of development that this variable be set at the end of operation completion. Would like to keep it that way.